_posts/2021-01-22-logarithmic-time-factoring-magic-trick.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Factoring in logarithmic time: a mathemagic trick"
+categories:
+author: Robin Jadoul
+meta: "Factoring in logarithmic time: a mathemagic trick"
+tags:
+excerpt_separator: <!--more-->
+---
+
+When you have an interest in cryptography, and you occasionally browse reddit, you've probably seen something like this before. Someone creates a new post on r/crypto or r/cryptography, claiming to be able to efficiently factor integers or RSA moduli. This then usually goes hand in hand with a serious lack of context and explanation, notation that is questionable at best, often just plain unreadable, and an aversion of providing proper proof by factoring an actual, cryptographically big modulus. Yet because it's hard to understand, and somethings there's math involved that at first glance seems like it might have some deeper meaning, you're left wondering, is there some insight to be found here.
```

Comment:
of providing proper proof -> to providing minimal sufficient proof (too many p's?)
somethings -> sometimes